tests/bluechi_test/test.py

```diff
@@ -116,6 +116,16 @@ def gather_logs(self, ctrl: BluechiControllerContainer, nodes: Dict[str, Bluechi
                 node.gather_valgrind_logs(self.tmt_test_data_dir)
 
         self.gather_test_executor_logs()
+    
+    def gather_coverage(self, ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer]):
+        logger.debug("Collecting coverage from all containers...")
+
+        if ctrl is not None:
+            ctrl.gather_coverage(self.tmt_test_data_dir)
+
+        for _, node in nodes.items():
+            logger.debug("!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!")
```

Comment:
Maybe worth changing to:

```python
logger.debug(f"Gathering coverage files from node '{node.node_name}'")
```

Reply:
This  was added for my debugging I will soon remove it.